views/index.jsx

```diff
@@ -17,6 +32,7 @@ function render({ slug, meta, html }) {
 				<p>{meta.subtitle}</p>
 				<p>{meta.description}</p>
 			</TitleBlock>
+			{renderBreadcrumbs(slug, meta.h1)}
```

Comment:
Just out of curiosity: Why isn't this a macro, i.e. avoiding JSX?

Reply:
I think the reason was that I already had the component `<Breadcrumb>` which I developed as a component and I didn't want to write a different `<Breadcrumb>` just to do the conditional rendering. It may not be a good reason.

I think for me a Macro is a "thing" and a function is an "action". Here I wanted to do an action "render breadcrumbs" so I reached for a function instead of a macro. But I think either would work.

Reply:
I occasionally find myself avoiding macros in favor of simple functions as well, so yeah, no objection. (Having said that, I'd consider breadcrumbs very much a _thing_ here.)